Tasks/ServiceFabricDeployV1/ServiceFabricSDK/Publish-UpgradedServiceFabricApplication.ps1

```diff
@@ -146,6 +146,32 @@ function Publish-UpgradedServiceFabricApplication
         throw $errMsg
     }
 
+    $ApplicationManifestPath = "$AppPkgPathToUse\ApplicationManifest.xml"
+
+    $names = Get-NamesFromApplicationManifest -ApplicationManifestPath $ApplicationManifestPath
+    if (!$names)
+    {
+        return
+    }
+
+    # If ApplicationName is not specified on command line get application name from Application parameter file.
```

Comment:
Is there a particular reason why you changed order to first get application names from manifest file and then get application name from parameter file?

Reply:
Both are different. I think order doesn't matter here.
Still ApplicationName variable is being used in fetching old application so placed this right before of it.